app/user_test.go

```diff
@@ -234,6 +234,7 @@ func TestUpdateActiveBotsSideEffect(t *testing.T) {
 	})
 
 	th.App.UpdateActive(th.BasicUser, false)
+	require.NotZero(t, th.BasicUser.DeleteAt)
```

Comment:
This just tests the user's deleteAt field, but does not test that the channel is deleted too. I would prefer a separate test where we create a user, and then create 2 DM channels with new users and then delete the user and verify that those DM channels have been marked as deleted.